notion-core/src/main/scala/com/ambiata/notion/core/Location.scala

```diff
@@ -27,7 +27,8 @@ sealed trait Location {
 
 case class HdfsLocation(path: String) extends Location {
 
-  def render: String = path
+  def render: String =
+    "hdfs://"+path
```

Comment:
It should not, it should be created with the `fromUri` method which strips out the scheme.